miasm2/analysis/mem.py

```diff
@@ -0,0 +1,1259 @@
+"""This module provides classes to manipulate C structures backed by a VmMngr
+object (a miasm VM virtual memory).
```

Comment:
VM stands for Virtual Machine here, I will change it to sandbox to avoid confusion, thanks.